grpc/server.go

```diff
@@ -66,14 +69,14 @@ func (sb *serverBuilder) Add(desc *grpc.ServiceDesc, impl any) *serverBuilder {
 // all of the services added to the builder. It also exposes a health check
 // service. It returns two functions, start() and stop(), which should be used
 // to start and gracefully stop the server.
```

Comment:
The documentation should be updated, both to remove the mention of stop(), and to describe the new shutdown behavior.

Reply:
Hah, I have a commit which does exactly this already, but it's on my desktop. I'll push it on monday.